docs/src/makie.md

```diff
@@ -0,0 +1,66 @@
+# Makie.jl plots
+
+Compared to Gadfly.jl and StatsPlots.jl, Makie.jl is the most flexible plotting library that you can use.
```

Comment:
Is this actually true? One can tweak and combine StatsPlots (or probably Gadfly, not sure?) also a lot. IMO the advantages of Makie are quality of plots (CairoMakie pdfs and svgs) and interactivity (eg GLMakie and WGLMakie). Probably shouldn't be discussed here. Can we change it to something more neutral and less judging?